frontend/src/concepts/pipelines/content/tables/utils.ts

```diff
@@ -139,3 +146,8 @@ export const isJobWithinDateRange = (
     (endNumber >= jobStartNumber && endNumber <= jobEndNumber)
   );
 };
+
+export const getPipelineJobExecutionCount = (resourceName: string) => {
+  const regex = /\w+(?=-[^-]*$)/;
+  return resourceName?.match(regex)?.[0];
+};
```

Comment:
Please write these on this -- there is a failure state here you're not watching.

Also type your response from this function.

Reply:
```
getPipelineJobExecutionCount('test')
undefined
getPipelineJobExecutionCount('test-5-something')
'5'
getPipelineJobExecutionCount('test-something')
'test'
```
There are other use-cases here -- I think you need to take a stance on what this is intended on doing. Because `test` is not a count.